### PR TITLE
MVS-532 Set up unit testing for broker

### PR DESCRIPTION
**Pull request by jeremylimson**
## :floppy_disk: [Code Check]
_Make sure you've checked/done the following before making this PR!_
* Does your code build/run?
* Does your design approach make sense?
* Does your work achieve the intention of the card?
* Does your code have any unintentional side effects -- have you tested a full workflow & checked the logic branches you touched?
* Did you make a unit test for your code (as appropriate)?
* Is your branch relatively up to date with the Development branch (within ~2 commits)
* Have you moved the card to "Needs QA" in Jira

## :flipper: [JIRA Sprint Card]
https://mass-immunization-system.atlassian.net/browse/MVS-532

## :newspaper: [Work Description]
Established general framework for unit testing toFHIR and toModel functions for Broker app

## :vertical_traffic_light: [Testing/QA Notes]
Unit test was conducted using mock data from Example.js. The test is passed if the results of toFHIR and toModel matched.

### CODE REVIEW

File: broker/models/test/AppointmentTest.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        let obj = JSON.parse(data.Appointment);
+        let input = obj.Appointment;
+        let fhir = app.toFHIR(input);
+
+        let output = app.toModel(fhir);
+        
+        for (let property in input) {
+            //console.log(input.property);
+          if (output.property !== input.property) {
+            console.log(property);
```
Comment (axn9t9) [inline broker/models/test/AppointmentTest.js:19]: Is there a more mocha/chai-ic method for displaying information about the test failing?

File: broker/models/test/AppointmentTest.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          }
+        }
+
+        /*
+        for (let property in output) {
+            console.log(output.property);
+        }
+        */
+
+        assert(result, 'Test failed');
```
Comment (axn9t9) [inline broker/models/test/AppointmentTest.js:32]: Is there a more mocha/chai-ic method of doing this?

File: broker/models/test/AppointmentTest.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+        let output = app.toModel(fhir);
+        
+        for (let property in input) {
+            //console.log(input.property);
+          if (output.property !== input.property) {
+            console.log(property);
+            console.log(`Expected ${input.property}; received ${output.property}`)
+            result = false;
+            //break;
```
Comment (axn9t9) [inline broker/models/test/AppointmentTest.js:22]: Should we break-early, immediately when the test fails?

**Review by axn9t9 — CHANGES_REQUESTED** [review]

File: broker/models/test/AppointmentTest.js (hunk 2 above)
Comment (axn9t9) [reply broker/models/test/AppointmentTest.js:32]: There are a lot of assertion methods - I think assert is fine.

File: broker/models/test/AppointmentTest.js (hunk 1 above)
Comment (jeremylimson) [reply broker/models/test/AppointmentTest.js:19]:
```suggestion
            assert.fail(`Expected ${input.property}; received ${output.property}`);
```
Throwing a failure should break from the test and display why the test failed

**Review by axn9t9 — APPROVED** [review]

**Pull request merged**